### ISSUE TITLE
FUNSD GeoLayoutLM EL experimental results cannot be reproduced

### ISSUE THREAD

**Issue by wang-weishi** [chongzhangFDU/ROOR#5]
Dear authors,

Thank you for your contribution. May I ask about my reproducing results in Table 3?
I can reproduce the FUNSD EL results for LayoutLMv3 base and LayoutLMv3 large (in Table 3). However, there is a huge gap between my reproduced GeoLayoutLM result and the reported result for the EL task. Could you please share the preprocessed FUNSD (it would be super helpful to share other preprocessed datasets or share the script), and your script for the GeoLayoutLM EL task? 

Below is my experimental setup, by following the provided bash file.
CUDA_VISIBLE_DEVICES=0 python tasks/train_geo.py \
  --do_train true \
  --image_dir FUNSD-Datasets/data \
  --json_dir FUNSD-Datasets/data/jsons \
  --split_file_dir ROOR-Datasets/data \
  --train_dataset_name data.train.txt \
  --valid_dataset_name data.val.txt \
  --test_dataset_name data.val.txt \
  --label_file_name labels.txt \
  --config_json_pathROOR/rore/configs/config-1024.json \
  --bert_base_path .cache/huggingface/models/models--bert-base-uncased/snapshots/86b5e0934494bd15c9632b12f734a8a67f723594 \
  --model_ckpt_path .cache/huggingface/models/geolayoutlm-large-1024/geolayoutlm_large_pretrain.pt \
  --save_model_dir saved_models/funsd/re/geolayoutlm-large-pretrain-1024-use-segment-re-no-layouts \
  --use_vision true \
  --use_segment true \
  --linking_coeff 0.5 \
  --lam 10 \
  --num_ro_layers 4 \
  --use_aux_ro false \
  --transitive_expand false \
  --batch_size 4 \
  --accumulate_grad_batches 4 \
  --max_epochs 500 \
  --learning_rate 1e-5 \
  --weight_decay 0.01 \
  --warmup_ratio 0.02 \
  --monitor val_linking_f1 \
  --log_every_n_steps 1 \
  --keep_checkpoint_max 1 \
  --patience 50 \
  --shuffle true \
  --seed 2024 \
  --gpus 1

**Comment by wang-weishi** [comment]
Adding more details, this refers to the GeoLayoutLM EL experiment without leveraging reading order information.

**Comment by chongzhangFDU** [comment]
1. The request to share the preprocessed datasets: The authors are sorry that according to the usage policies of the datasets mentioned in the paper, we are not permitted to publish our preprocessed versions of these datasets. 
If you wish to use RORE for custom datasets, you can train a ROP model based on the code in the `rop/` directory and the published [ROOR dataset](https://github.com/chongzhangFDU/ROOR-Datasets). Then, you can use this model to generate pseudo reading order relation labels for your custom datasets. 
2. The issue of being unable to reproduce the result of FUNSD-GeoLayoutLM-EL: Please attempt to configure the `linking_coeff`. For the EL task, setting `linking_coeff` to 1 would be better. You may also increase the learning rate and the number of epochs for better performance.
